lemur/pending_certificates/views.py

```diff
@@ -213,6 +213,7 @@ def get(self, pending_certificate_id):
     @validate_schema(
         pending_certificate_edit_input_schema, pending_certificate_output_schema
     )
+    @operator_permission.require(http_exception=403)
```

Comment:
we can gate put with ADMIN_ONLY_CERTIFICATE_CREATION property, I don't see a strong reason for delete / post to be gated though